src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -53,7 +53,7 @@ const LoginForm: React.FunctionComponent<LoginFormProps> = ({
     reValidateMode: "onChange",
   });
 
-  if (!space) return null;
+  if (!space || !world) return null;
```

Comment:
Maybe a loading screen, message (we got the components for it), but if hard to determine which is appropriate even `null` is OK, just maybe with `@debt` so we figure it out later